torch/csrc/api/include/torch/ordered_dict.h

```diff
@@ -221,14 +225,14 @@ class OrderedDict<Key, Value>::Item {
   }
 
   /// Returns a `(key, value)` pair.
-  const std::pair<const Key, Value>& pair() const noexcept {
+  std::pair<const Key, Value> pair() const noexcept {
```

Comment:
Curious: do we have to change the return type to `std::pair<const Key, Value>` for the copy-assignment operator to work?